lib/project_types/script/ui/error_handler.rb

```diff
@@ -102,6 +102,18 @@ def self.error_messages(e)
               e.extension_point_type
             ),
           }
+        when Layers::Domain::Errors::ServiceFailureError
+          {
+            cause_of_error: ShopifyCli::Context.message('script.error.service_failure_cause'),
+          }
+        when Layers::Domain::Errors::MetadataValidationError
```

Comment:
This should have a help suggestion specifying to check that their `@shopify/scripts-toolchain-as` NPM package and the extension point NPM package they are using for their script is up-to-date.